manual/Makefile

```diff
@@ -309,3 +309,11 @@ fixmes:
 		echo -n "Total FIXMES: " && grep "FIXME{" $$i | wc -l; \
 		echo; \
 	done
+
+##############################################################################
+# Instructions to manually generate glossary
+	@echo "FIXME: Add script to generate Glossary to Makefile"
+	@echo "====> To generate glossary, run:"
+	@echo "              makeindex -s manual.ist -o manual.gls manual.glo"
+	@echo "              make
+	@echo "              touch latex.tex"
```

Comment:
This should be part of the Makefile and shouldn't need to be done manually. That way it can't be forgotten either.

Is there any new dependency introduced by glossaries? If so, they should be added to the README.md. I noticed when trying to build the manual that the dependency on `roboto.sty` is missing, haven't tried your branch yet.

I'll do the Makefile change if you want, but may need your help. It seems make already needs to run twice to get references rights, so all we need to do is adding the `makeindex -s manual.ist -o manual.gls manual.glo` somewhere.

Reply:
@Indanz I mentioned making consistent capitalisation in the description of this PR, but should have explicitly pointed out that it was for CPtr. Should I do a separate PR for consistent naming of CPtr for clarity?

I should have also pointed out that the FIXME in the Makefile is a temporary instruction to generate manual.pdf. The aim was to include the glossary generation as part of the Makefile once I figured out how to do it, and then remove the temporary instructions. I'll fix it now.

Reply:
> I mentioned making consistent capitalisation in the description of this PR

Hmm, maybe something went wrong, I see no description at all for this PR, hence my surprise seeing other changes than the ones mentioned in the title.

Separate PR would be better in this case because both commits change many lines and are quite independent, and it would make it possible to merge one before the other, but it's not crucial, both are manual improvements and in that sense related.

Reply:
@Indanz 

> I mentioned making consistent capitalisation in the description of this PR
> 
> Hmm, maybe something went wrong, I see no description at all for this PR, hence my surprise seeing other changes than the ones mentioned in the title.

Yes something went wrong - I put "consistent capitalisation" in the commit message, not the description. Sorry :-/

> Separate PR would be better in this case because both commits change many lines and are quite independent, and it would make it possible to merge one before the other, but it's not crucial, both are manual improvements and in that sense related.

Have created a separate PR: https://github.com/seL4/seL4/pull/1146